src/components/BlogCard/BlogCard.react.js

```diff
@@ -0,0 +1,107 @@
+//@flow
+
+import * as React from "react";
+import Card from "../Card/Card.react";
+import Icon from "../Icon/Icon.react";
+
+type Props = {|
+  +children?: React.Node,
+  +title?: string,
+  +titleUrl?: string,
+  +avatarUrl?: string,
+  +description?: string,
+  +date?: string,
+  +imgUrl?: string,
+  +iconName?: string,
+  +iconHref?: string,
+  +authorName?: string,
+  +avatarImgSrc?: string,
+  +imgSrc?: string,
+  +imgAlt?: string,
+  +aside?: boolean,
+  +postHref?: string,
+|};
+
+function BlogCard({
+  children,
+  title,
+  description,
+  avatarUrl,
+  imgUrl,
+  imgAlt,
+  aside,
+  authorName,
+  date,
+  imgSrc = " ",
```

Comment:
I'm curious why is the default for `imgSrc` and `avatarImgSrc` is a space and not `""`?